docker-compose.yml

```diff
@@ -1,65 +1,109 @@
 version: '3.9'
 
-x-base_service: &base_service
-    ports:
-      - "7860:7860"
-    volumes:
-      - &v1 ./data:/data
-      - &v2 ./output:/output
-    stop_signal: SIGINT
-    deploy:
-      resources:
-        reservations:
-          devices:
-              - driver: nvidia
-                device_ids: ['0']
-                capabilities: [gpu]
+x-base_service:
+  &base_service
+  ports:
+    - "7860:7860"
+  volumes:
+    - &v1 ./data:/data
+    - &v2 ./output:/output
+  stop_signal: SIGINT
+  deploy:
+    resources:
+      reservations:
+        devices:
+          - driver: nvidia
+            device_ids: [ '0' ]
+            capabilities: [ gpu ]
+
+x-base_service_amd:
+  &base_service_amd
+  ports:
+    - "7860:7860"
+  volumes:
+    - &v1 ./data:/data
+    - &v2 ./output:/output
+  stop_signal: SIGINT
+  group_add:
+    - video
+  devices:
+    - "/dev/dri"
```

Comment:
this does not work on a Mac

Reply:
@zthxxx linux only, I don't think this works for mac or windows either?
If someone actually figures out how to do this AMD setup with docker in windows, that would be really really awesome.

Reply:
No idea, no mac or windows handy to try with either.